x-pack/plugins/security_solution/public/detections/components/alerts_table/actions.tsx

```diff
@@ -4,8 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+/* eslint-disable complexity */
```

Comment:
Oooweee! Did we finally make it? Haha 😅 

I removed this locally and wasn't running into the error, so perhaps we're still good? Either way, I think the Alert Actions code has grown enough now that we can probably restructure and break it apart a bit further. This area of the codebase is only going to grow more and more now that we can programmatically declare actions on a per alert basis... 🙂

I've added this to my list of tech debt items for 7.10+, so nothing to worry about now.

<p align="center">
  <img width="250" src="https://user-images.githubusercontent.com/2946766/87350954-d95da700-c515-11ea-87ef-a7ff8ccd1f3c.gif" />
</p>